### PR TITLE
Homework2: Denis Borisenko

### PR DESCRIPTION
**Pull request by dbarysenka**
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle)
- [x] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks
- [x] my PR made from the fork feature branch to maxim homework# branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore

### CODE REVIEW

File: src/homework2/task2.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+"""Найти самое длинное слово в введенном предложении.
+   Учтите что в предложении есть знаки препинания.
+"""
+
+string = input('Введите предложение: ')
```
Comment (belovmd) [inline src/homework2/task2.py]: don't solve task in description

File: src/homework2/task5.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+"""Выведите n-ое число Фибоначчи, используя только временные переменные,
+   циклические операторы и условные операторы. n - вводится.
+"""
+
+n = int(input('Введите n-ое число Фибоначчи: '))
+number1 = 0
```
Comment (belovmd) [inline src/homework2/task5.py:6]: 1, 1 - start values

**Review by belovmd — CHANGES_REQUESTED** [review]
you can add 2 additional advanced tasks for higher score

**Comment by dbarysenka** [comment]
hw

**Review by dbarysenka — COMMENTED** [review]
w

**Review by belovmd — COMMENTED** [review]
8,5

File: src/homework2/task7.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    up_arr = []
+
+    for i in str(num):
+        up_arr.append(int(i))
+
+    return up_arr
+
+
+""" # 2Напишите функцию, которая принимает массив из 10 целых чисел(от 0 до 9)
+   который возвращает строку этих чисел в форме номера телефона.
```
Comment (belovmd) [inline src/homework2/task7.py:86]: easy task for python

**Pull request closed without merging**